lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -934,7 +934,11 @@ sub asana_req {
     $req->header("x-hub-signature" => $header_data);
     $req->content($json_data);
 
-    my $result = $ua->request($req);
+    my $result;
+    try{
+        $result = $ua->request($req);
```

Comment:
We already have a LWP::UA instance available in `$c->d->http`, no need to instantiate new ones.